configs/wf_configs/cp_process.yaml

```diff
@@ -48,22 +49,24 @@ annotate_configs:
 
 aggregate_configs:
   params:
-  strata:
-    - Metadata_Plate
-    - Metadata_Well
-  features: infer
-  operation: median
-  output_file: none
-  compute_object_count: False
-  object_feature: Metadata_ObjectNumber
-  subset_data_df: none
-  compression_options:
-    method: gzip
-    mtime: 1
-  float_format: null
+    input_data: annotated
+    strata:
+      - Metadata_Plate
+      - Metadata_Well
+    features: infer
+    operation: median
+    output_file: none
+    compute_object_count: False
+    object_feature: Metadata_ObjectNumber
+    subset_data_df: none
+    compression_options:
+      method: gzip
+      mtime: 1
+    float_format: null
 
 normalize_configs:
   params:
+    input_data: aggregated
```

Comment:
Where do the input_data values come from? Would it make sense to document these somewhere (if this hasn't already been done)? A data-based approach might entail creating a yaml schema which might help ensure all expected fields exist and are populated as expected (for example, [jsonschema](https://python-jsonschema.readthedocs.io/en/stable/faq/#can-jsonschema-be-used-to-validate-yaml-toml-etc)).

Reply:
Good point. I'll refer this comment to this issue #55 . Currently the configs are going through rapid changes. Greg and I discussed this considered this to be a hot fix for `multi-paramter` modules. 

in addition, documentation added!